mozilla_pipeline_schemas/cli/commands.py

```diff
@@ -0,0 +1,10 @@
+import click
+from .bigquery import bigquery
+
+
+@click.group()
+def entry_point():
+    pass
+
+
+entry_point.add_command(bigquery)
```

Comment:
This can be removed since the command group is already created in `cli/__init__.py`

Reply:
Looks like that accidentally got checked in (or I forgot to remove it when shuffling things around).